test/battle/ability/dancer.c

```diff
@@ -0,0 +1,35 @@
+#include "global.h"
+#include "test/battle.h"
+
+SINGLE_BATTLE_TEST("Dancer can copy a dance move immediately after it was used addionally to the users move")
```

Comment:
```suggestion
SINGLE_BATTLE_TEST("Dancer can copy a dance move immediately after it was used and allow the user of Dancer to still use its move")
```
Does this capture the intent of test better?

Reply:
it does, ye